src/lava/proc/lif/models.py

```diff
@@ -69,7 +69,7 @@ class AbstractPyLifModelFixed(PyLoihiProcessModel):
     v: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=24)
     du: int = LavaPyType(int, np.uint16, precision=12)
     dv: int = LavaPyType(int, np.uint16, precision=12)
-    bias_mant: np.ndarray = LavaPyType(np.ndarray, np.int16, precision=13)
+    bias_mant: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=13)
```

Comment:
This is a bit unintuitive. We only "allow" 13-bit precision but need the 32 bits to do shifts later on. I know that leaving it at 16-bit is not much better...
But maybe the 32-bit variable could be introduced as a local copy of `bias_mant` inside the `scale_bias` method? That makes it more obvious what it is needed for. And I would put a comment on it to highlight that the 32 bits are important at the point.

Reply:
I agree that this is unintuitive.
I followed your suggestion of introducing a local copy with promoted ```dtype```, see 0b2c64b.